src/lib/components/profile/carousel/Carousel.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+  import rightArrow from '$lib/icons/right-arrow.svg'
+  import leftArrow from '$lib/icons/left-arrow.svg'
+
+  export let images: string[]
+
+  let currentImage = 0
+
+  let nextImage = 1
+
+  let prevImage = images.length - 1
+
+  function goNext() {
+    currentImage = (currentImage + 1) % images.length
+  }
+
+  function goPrev() {
+    currentImage = (currentImage - 1 + images.length) % images.length
+  }
+
+  $: nextImage = (currentImage + 1) % images.length
+  $: prevImage = (currentImage - 1 + images.length) % images.length
+</script>
+
+<div class="flex flex-col items-center justify-center gap-4">
```

Comment:
```suggestion
<section class="flex flex-col items-center justify-center gap-4">
```
Eso para mí es un section xd